versions/1.21.5/src/main/java/at/hannibal2/skyhanni/features/hunting/FusionKeybinds.kt

```diff
@@ -0,0 +1,55 @@
+package at.hannibal2.skyhanni.features.hunting
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.config.features.hunting.HuntingConfig
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.GuiKeyPressEvent
+import at.hannibal2.skyhanni.events.SecondPassedEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.KeyboardManager.isKeyHeld
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import org.lwjgl.glfw.GLFW
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object FusionKeybinds {
+
+    val config get() = SkyHanniMod.feature.hunting.fusionKeybinds
+
+    private var lastDuplicateKeybindsWarnTime = SimpleTimeMark.farPast()
+
+    @HandleEvent(onlyOnIsland = IslandType.GALATEA)
+    fun onKeybind(event: GuiKeyPressEvent) {
+        when (InventoryUtils.openInventoryName()) {
```

Comment:
Use repo patterns for the inventory names, and then InventoryDetectors

```suggestion
        val fusionBoxInventoryDetector = InventoryDetector(fusionBoxInventoryPattern)
        
        
        when {
        	fusionBoxInventoryDetector.isInside() ->
```

Reply:
> Use repo patterns for the inventory names, and then InventoryDetectors

dont think this is quite needed in this situation